### DISCUSSION TITLE
Need for a repo that guides beginners for contributing in Open Source

### DISCUSSION THREAD

**Discussion by rjcli** [AkashSingh3031/The-Complete-FAANG-Preparation#359] [Ideas]
I think there is a need for a repo that guides beginners (and me of course) in contributing to open source. 

Got this idea by visiting this great repo. Hope anyone starts a repo like this for open source too.

**Comment by AkashSingh3031** [comment]
@rashmiraj513 
Which type of guides of repo

if you want to know how to contribute then check this [link](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation#contributing-)
